doc/lua_api.md

```diff
@@ -4283,7 +4284,7 @@ After creating the `locale` directory, a translation template for the above
 example using the following command:
 
 ```sh
-xgettext -L lua -kS -kNS:1,2 -kminetest.translate:1c,2 -kminetest.translate_n:1c,2,3 \
```

Comment:
Should this have been left unchanged?

Reply:
I don't think so, it'd be weird suggesting `core` in the whole document and then provide an example looking for the deprecated alias